specs/spark-application.yaml

```
@@ -0,0 +1,38 @@
+apiVersion: "sparkoperator.k8s.io/v1beta1"
```

Let’s update the api version to /v1beta2 like recommended  [here](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator#project-status)
```suggestion
apiVersion: "sparkoperator.k8s.io/v1beta2"
```